pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/KitRequestShipping.java

```diff
@@ -751,14 +757,19 @@ public static Map<String, List<KitRequestShipping>> getAllKitRequestsByRealm(@No
     }
 
     private static PreparedStatement getPreparedStatement(@NonNull Connection conn, String target, @NonNull String realm, String type,
-                                                          boolean getAll) throws SQLException {
+                                                          Integer kitTypeId, boolean getAll) throws SQLException {
```

Comment:
Perhaps the `type` parameter should be name `kitTypeName`, now that we have a `kitTypeId`? `type` is not a good parameter name anyway, unless type is very obvious.